### PR TITLE
more specials mappings

### PR DESCRIPTION
**Pull request by winterbird-code**
| AniDB | TVDB/TMDB/IMDB | Notes |
| --- | --- | --- |
| https://anidb.net/anime/15 | https://thetvdb.com/index.php?tab=series&id=79107 | Unmap (as in map to 0) specials that doesn't exist in tvdb |
| https://anidb.net/anime/2304 | https://thetvdb.com/index.php?tab=series&id=129741 | Unmap special that doesn't exist in tvdb |
| https://anidb.net/anime/3067 | https://thetvdb.com/index.php?tab=series&id=80759 | Fix specials mapping |
| https://anidb.net/anime/4325 | https://thetvdb.com/index.php?tab=series&id=81144 | Add specials mapping |
| https://anidb.net/anime/15964 | https://thetvdb.com/index.php?tab=series&id=402117 | Add specials mapping |

### CODE REVIEW

File: anime-list-master.xml [hunk 1]
```diff
@@ -74,6 +74,9 @@
   </anime>
   <anime anidbid="15" tvdbid="79107" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Kanon</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-1;2-0;3-0;4-0;</mapping>
+    </mapping-list>
```
Comment (BrutuZ) [inline anime-list-master.xml:79]: Same season 1-1(;2-2;3-3;...) is implied, no need to specify it.
TVDB only has one special episode so the AniDB ones won't conflict with anything, there's no need to unmap them.
```suggestion

```

File: anime-list-master.xml [hunk 2]
```diff
@@ -7826,6 +7829,9 @@
   </anime>
   <anime anidbid="2304" tvdbid="129741" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Perrine Monogatari</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-0;</mapping>
+    </mapping-list>
```
Comment (BrutuZ) [inline anime-list-master.xml:7834]: Unmapping without conflict
```suggestion

```

File: anime-list-master.xml [hunk 3]
```diff
@@ -9878,7 +9884,7 @@
   <anime anidbid="3067" tvdbid="80759" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Ichigo 100%</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;3-0;4-0;5-0;</mapping>
+      <mapping anidbseason="0" tvdbseason="0">;1-1;2-6;3-0;4-0;5-0;</mapping>
```
Comment (BrutuZ) [inline anime-list-master.xml:9887]: 1-1 is implied
```suggestion
      <mapping anidbseason="0" tvdbseason="0">;2-6;3-0;4-0;5-0;</mapping>
```

File: anime-list-master.xml [hunk 4]
```diff
@@ -12740,6 +12746,9 @@
   </anime>
   <anime anidbid="4325" tvdbid="81144" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>True Tears</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0" start="2" end="8" offset="-1">;1-0;9-9;10-0;11-0;12-0;13-8;</mapping>
```
Comment (BrutuZ) [inline anime-list-master.xml:12750]: `start="1"` will unmap episode 1 by the offset itself 👍 
```suggestion
      <mapping anidbseason="0" tvdbseason="0" start="1" end="8" offset="-1">;10-0;11-0;12-0;13-8;</mapping>
```

File: anime-list-master.xml [hunk 5]
```diff
@@ -43948,6 +43957,9 @@
   </anime>
   <anime anidbid="15964" tvdbid="402117" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Kawaii dake ja Nai Shikimori-san</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-4;2-0;3-0;4-5;5-0;</mapping>
```
Comment (BrutuZ) [inline anime-list-master.xml:43961]: While there's no conflict for the other episodes, these unmappings don't feel as out of place given the "hole" on the TVDB numbering (why on Earth their specials start at 04?! Crazy people) and the show being recent enough for either release new OVAs or TVDB mods freaking out on ordering as they often do.

**Review by BrutuZ — COMMENTED** [review]
Unmapping is usually to solve conflicts, for example: Season 1 Special 1 from AniDB doesn't exist on TVDB, Season 2 Special 1 does as S00E01, Special 1 on both AniDB entries are naturally mapped to TVDB's S00E01, so we unmap the first one.

**Comment by winterbird-code** [comment]
> Unmapping is usually to solve conflicts, for example: Season 1 Special 1 from AniDB doesn't exist on TVDB, Season 2 Special 1 does as S00E01, Special 1 on both AniDB entries are naturally mapped to TVDB's S00E01, so we unmap the first one.

Ah, I interpreted "conflict" broader as "If the episode in anidb does not match any episode in tvdb". Probably because that's what I actually needed :flushed: 

Not unmapping these specials cause a bit of a problem for me as my use case involves using this list to sort series, and episodes, to a tvdb compatible library if, and only if, tvdb has metadata for it. The tool only has access to the anidb API and this list, so without the unmapping I have no way of knowing if a special episode is *not* present in tvdb and thus should not be sorted to the tvdb library. I'll have to think about this :thinking:

File: anime-list-master.xml (hunk 5 above)
Comment (winterbird-code) [reply anime-list-master.xml]: OK, I'll keep this mapping then.

**Pull request merged**